### PR TITLE
Fix CodeBuild test failures

### PR DESCRIPTION
**Pull request by srherbener**
Initially, this PR is here just to debug the CodeBuild test failures. If it turns out that a code change to nc2odb.py is necessary, then this will provide those changes.

### CODE REVIEW

**Comment by mer-a-o** [comment]
@srherbener, I was able to recreate the error in the intel17 container. It might be easier/faster to debug in the container than on CodeBuild.

**Comment by srherbener** [comment]
@mer-a-o, great! Thank you for testing the container.

I was starting to get the impression that the develop branch was being tested instead of the bugfix/cbuild-tests branch. I was placing print statements in the scripts and I could see the messages when run on my laptop and did not see the messages in the CodeBuild test. Is it possible that this could be happening?

I'll try the intel container since you got the problem reproduced there. That sounds much easier.

Thanks

**Comment by mer-a-o** [comment]
@srherbener Yes! You are right, it tests the develop branch. This is why we did not see the test failures before merging the odc branch into develop. I updated CodeBuild to test the correct branch. Thanks for catching this bug.

**Comment by srherbener** [comment]
@mer-a-o, thanks for getting this fixed! I'll remove the label and submit this PR for review/merge.

File: src/odc/nc2odb.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def DtToDateTime(Dt):
 
 # Write the header
 Line = ['DuMmYiNdEx@MetaData:INTEGER']
-for VarName in VarList:
+for VarName in sorted(VarList):
```
[the remark this answers is not in the document]
Comment (srherbener) [reply src/odc/nc2odb.py]: @svahl991, good idea. I went ahead and created the SortedVarList variable.

**Pull request merged**